### PR TITLE
Add tutorial to docs

### PR DESCRIPTION
**Pull request by anonymousr007**
Fixes [#86](https://github.com/nasa/prog_algs/issues/86)

### CODE REVIEW

File: sphinx_config/index.rst [hunk 1]
```diff
@@ -7,9 +7,13 @@ The Prognostics Models Package was developed by researchers of the NASA Prognost
...
+   :target: https://mybinder.org/v2/gh/nasa/prog_models/dev?labpath=tutorial.ipynb
+
 .. toctree::
    :maxdepth: 2
    
+   Tutorial <https://mybinder.org/v2/gh/nasa/prog_models/dev?labpath=tutorial.ipynb>
```
Comment (teubert) [inline sphinx_config/index.rst:16]:
```suggestion
   Tutorial <https://mybinder.org/v2/gh/nasa/prog_algs/dev?labpath=tutorial.ipynb>
```
This should point to the prog_algs tutorial

File: sphinx_config/index.rst [hunk 2]
```diff
@@ -7,9 +7,13 @@ The Prognostics Models Package was developed by researchers of the NASA Prognost
...
+   :target: https://mybinder.org/v2/gh/nasa/prog_models/dev?labpath=tutorial.ipynb
```
Comment (teubert) [inline sphinx_config/index.rst:11]:
```suggestion
   :target: https://mybinder.org/v2/gh/nasa/prog_algs/dev?labpath=tutorial.ipynb
```
This should point to prog_algs tutorial

**Review by teubert — CHANGES_REQUESTED** [review]
Thanks! Just a small change requested to point to prog_algs tutorial instead of prog_models

**Comment by anonymousr007** [comment]
I will change all just a minute

**Comment by anonymousr007** [comment]
It's done.

**Comment by teubert** [comment]
Thanks @anonymousr007

**Pull request merged**